<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Canny</class>
 <widget class="QDialog" name="Canny">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>第一个滞后性阈值</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>第二个滞后性阈值</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Sobel算子的孔径大小</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox">
     <property name="maximum">
      <double>255.000000000000000</double>
     </property>
     <property name="value">
      <double>150.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
     <property name="maximum">
      <double>255.000000000000000</double>
     </property>
     <property name="value">
      <double>100.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
